.travis.yml

```diff
@@ -7,14 +7,19 @@ os:
 compiler:
   - gcc
   - clang
-  
+
+env:
+  - USE_CMAKE=YES
+  - USE_CMAKE=NO
+
 addons:
   apt:
     packages:
       - texinfo
 
 script:
-  - autoreconf -i
-  - ./configure
+  - mkdir build
+  - cd build
+  - ../scripts/run_config.sh
```

Comment:
A small thing, but as the number of lines necessary to build is small, could it be included directly in the .travis.yml file instead of as a separate script?